TeamCode/src/main/java/org/firstinspires/ftc/teamcode/modules/robot/Claw.java

```diff
@@ -1,41 +1,59 @@
 package org.firstinspires.ftc.teamcode.modules.robot;
 
 import com.qualcomm.robotcore.hardware.Servo;
+import com.qualcomm.robotcore.util.ElapsedTime;
 
 import org.firstinspires.ftc.robotcore.external.Telemetry;
 import org.firstinspires.ftc.teamcode.opmode.config.ClawConfig;
 
 public class Claw extends BaseComponent {
-    static final float TOP_CLOSE = 1;
-    static final float TOP_OPEN = 0.5f;
+    static final float TOP_CLOSE = 0;
+    static final float TOP_OPEN = 0.35f;
     static final float SIDE_CLOSE = 0;
-    static final float SIDE_OPEN = 0.1f;
+    static final float SIDE_OPEN = 0.25f;
 
     Servo top;
     Servo side;
 
+    int scoringStep = 0;
+    ElapsedTime elapsedTime;
+
+    boolean scoring = false;
+
     public Claw(ClawConfig config) {
         this.top = config.getTop();
         this.side = config.getSide();
 
         top.setDirection(Servo.Direction.FORWARD);
         side.setDirection(Servo.Direction.FORWARD);
+
+        top.setPosition(TOP_CLOSE);
```

Comment:
This will move the servo on init, is this what you want?  My suggestion to avoid penalty, is to NOT move these on creation/initialization of component but rather explicitly call it in its own init() method.  That way you have total control over when it inits.